ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -8,6 +8,14 @@ declaration template components/dpmlfc/schema;
 
 include { 'quattor/schema' };
 
+function component_dpmlfc_number_string_valid = {
+  if ( !match(ARGV[0],'^\d+$') ) {
```

Comment:
Unfortunately `is_long` tells you if the variable is a `long` and will return false is you pass it a string... Here the problem is that one resource contains 2 elements, one of them being a string. Thus the other one is a string from pan point of view but has to be interpreted as a number. This validation function ensures that the string is a valid number.